src/fee_lab_to_nwb/scherrer_ophys/convert_session.py

```diff
@@ -52,9 +52,15 @@
     ),
 )
 
-timestamps = get_timestamps_from_csv(file_path=behavior_data_file_path)
+unadjusted_timestamps = get_timestamps_from_csv(file_path=behavior_data_file_path)
```

Comment:
Just a minor thing for readability, but I saw `unadjusted_timestamps` as corresponding to `adjust_to_zero=False`, whereas this is the one `timestamps` variable that does seem adjusted to zero

Reply:
I just fixed that with using two functions instead of one.